js/giphy.js

```diff
@@ -0,0 +1,38 @@
+$(document).ready(function(){
+
+  $("button").click(function(evt){
+    evt.preventDefault();
+    var api = "http://api.giphy.com/v1/gifs/search?&api_key=dc6zaTOxFJmzC";
+    var search = $("#keyword").val();
+    var q = "&q=" + search;
+    var url = api + q;
+    var count = 0;
+    var i = 0;
```

Comment:
@beckybeauchamp1 This code looks good for the most part, but I can't quite tease out the meaning / use of these two vars (`i` and `count`). I'd suggest re-naming them to be a bit more meaningful!